src/pygeoprocessing/geoprocessing.py

```diff
@@ -2554,7 +2554,6 @@ def _fill_work_queue():
 
     # this set is indexed by (x,y) tuple from iterblock offsets to check if the
     # target block is on its first pass and set to zero if so.
```

Comment:
Can this comment go as well?

Reply:
Yes, removed in 8afbb2e387c996f4905c3edf02a6802596e2c334